ci-operator/step-registry/telcov10n/performance/telcov10n-performance-commands.sh

```
@@ -0,0 +1,65 @@
+      vpn_password: "${VPN_PASSWORD}"
+      vpn_url: "${VPN_URL}"
+      tun_name: "${IFNAME}"
+    bastion:
+      ansible_host: "${BASTION_ADDRESS}"
+      ansible_user: "${BASTION_USERNAME}"
+      ansible_ssh_private_key_file: "${SSH_KEY}"
+      ansible_ssh_common_args: '${SSHOPTS[*]} -o ProxyCommand="ssh -W %h:%p ${SSHOPTS[*]} -q ${JUMP_SERVER_USERNAME}@${JUMP_SERVER_ADDRESS}"'
+END_INVENTORY
+
+ansible-galaxy collection install ansible.posix
+ansible-playbook -i inventory.yml playbooks/run_performance_benchmark.yml -e "artifact_dir=${ARTIFACT_DIR}" -v | tee ${ARTIFACT_DIR}/ansible.log
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)